scripts/configure_jenkins.sh

```diff
@@ -74,6 +74,7 @@ generate_inline_script_job $SCRIPTS_DIR/../jobs/openshift/cluster/brew/openshift
 # OpenShit 4 Cluster jobs
 jenkins-jobs --conf $CONFIG update $SCRIPTS_DIR/../jobs/openshift4/cluster/create/openshift4-cluster-create.yaml
 jenkins-jobs --conf $CONFIG update $SCRIPTS_DIR/../jobs/openshift4/cluster/deprovision/openshift4-cluster-deprovision.yaml
+jenkins-jobs --conf $CONFIG update $SCRIPTS_DIR/../jobs/openshift4/cluster/integreatly/openshift4-cluster-integreatly-install.yaml
```

Comment:
:+1: Added here https://integreatly-qe-jenkins.rhev-ci-vms.eng.rdu2.redhat.com/view/all/job/openshift4-cluster-integreatly-install

Have no cluster to try it against currently.

Reply:
@StanleyKaleta Actually, this should be:

```generate_inline_script_job $SCRIPTS_DIR/../jobs/openshift4/cluster/integreatly/openshift4-cluster-integreatly-install.yaml```